react/inputs/StyleguideInput/AutocompleteInput.js

```diff
@@ -29,7 +30,7 @@ class AutocompleteInput extends Component {
           inputValue,
           highlightedIndex,
         }) => (
-            <div className="vtex-profile-form__autocomplete-input-wrapper relative">
+            <div className={`${styles.styleguideInput} ${styles.autocompleteInput} relative`}>
```

Comment:
why this one has two classes?

Reply:
Since since this one is a different component of the rest, you may want to put some different styling in some case.